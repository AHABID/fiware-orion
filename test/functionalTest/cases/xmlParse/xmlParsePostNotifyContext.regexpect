HTTP/1.1 200 OK
Content-Length: 141
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<notifyContextResponse>
  <responseCode>
    <code>200</code>
    <reasonPhrase>OK</reasonPhrase>
  </responseCode>
</notifyContextResponse>
++++++++++++++++++++
