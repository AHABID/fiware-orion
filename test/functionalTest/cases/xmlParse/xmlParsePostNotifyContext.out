HTTP/1.1 200 OK
Content-Length: 141
Content-Type: application/xml
Date: Sun, 01 Jun 2014 17:45:39 GMT

<?xml version="1.0"?>
<notifyContextResponse>
  <responseCode>
    <code>200</code>
    <reasonPhrase>OK</reasonPhrase>
  </responseCode>
</notifyContextResponse>
++++++++++++++++++++
